src/fluidsynth_priv.h

```diff
@@ -84,6 +84,14 @@ typedef signed short       sint16;
 typedef unsigned short     uint16;
 typedef signed int         sint32;
 typedef unsigned int       uint32;
+typedef signed long long   sint64;
+typedef unsigned long long uint64;
+
+#ifdef __LP64__
```

Comment:
this should be `#if defined(__LP64__) || defined(_WIN64)` because win64 is LLP64 not LP64.